related_website_sets.JSON

```diff
@@ -1,5 +1,17 @@
 {
   "sets": [
+    {
+       "contact": "edward.goodnow@gmail.com",
+       "primary": "https://pennyauctionsoftware.net",
+       "associatedSites": [
+          "https://www.pennyauctionsoftware.net",
+          "https://demo.pennyauctionsoftware.net"
+       ],
+       "rationaleBySite": {
+          "https://www.pennyauctionsoftware.net": "https://www.pennyauctionsoftware.net",
+          "https://demo.pennyauctionsoftware.net": "https://demo.pennyauctionsoftware.net"
```

Comment:
It looks like this set is for testing purposes. We recommend using the local testing capabilities for RWS instead: https://developers.google.com/privacy-sandbox/blog/first-party-sets-testing-instructions#how_to_test_locally.